romac4j/src/main/java/com/rakuten/rit/roma/romac4j/connection/SocketPoolFactory.java

```diff
@@ -8,16 +8,18 @@
 public class SocketPoolFactory implements PoolableObjectFactory<Connection> {
     private String nodeId;
     private int bufferSize;
+    private int timeout;
 
-    public SocketPoolFactory(String nid, int bufferSize) {
+    public SocketPoolFactory(String nid, int bufferSize, int timeout) {

```

Comment:
thanks, 
I had updated.
and please also help to review another pull request
https://github.com/roma/roma-java-client-lite/pull/5